.github/workflows/lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Run linting
+
+on:
+    # Run on all pushes and on all pull requests.
+    # Prevent the "push" build from running when there are only irrelevant changes.
+    push:
+        paths-ignore:
+            - "**.md"
+    pull_request:
+    # Allow manually triggering the workflow.
+    workflow_dispatch:
+
+jobs:
+    test:
```

Comment:
Probably want a different name for this job.

Reply:
Honestly hadn't occurred to me `test` was variable. Is there a preferred convention here or just the slug'd name, e.g. `run_js_linting`?

Reply:
I'm not sure the [job_id](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) is actually exposed anywhere, as the "name" is shown instead, so as long as it's unique for the `jobs`, then whatever you want is fine; in this case though, calling it `test` and not actually running any tests didn't make sense to me.